### DISCUSSION TITLE
Welcome to home-assistant-idiamant Discussions!

### DISCUSSION THREAD

**Discussion by clementprevot** [clementprevot/home-assistant-idiamant#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lesloulousdu49** [comment]
Bonjour Clément.

Je viens vers toi car ton post m'intéresse car je suis intégrer avec une IDIAMENT et des voelts bubb.

J'ai installé ton .ZIP mais  qunad je veux l'intégrer je n'y arrive pas, pourrais tu m'aider ?

Message Erreur :
Missing code or state parameter in https://l.....duckdns.org:8123/auth/external/callback?error=invalid_client&state=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmbG93X2lkIjoiMjNhMDNiNzQxYzBhNjE0NDg5OTZkM2I5MzFlNzEzYTQiLCJyZWRpcmVjdF91cmkiOiJodHRwczovL2xlc2xvdWxvdXM0OS5kdWNrZG5zLm9yZzo4MTIzL2F1dGgvZXh0ZXJuYWwvY2FsbGJhY2sifQ.XtHndUZG7h1tq7dCbceT3ZF7sFaKyS-I9eDCbCj-0v8

En te remerciant pour ton aide 
Anthony

**Reply by clementprevot** [reply]
Salut Anthony,

Désolé l'intégration n'est pas du tout prête à être utiliser encore, je n'ai pas eu beaucoup de temps pour bosser dessus 😞

**Reply by lesloulousdu49** [reply]
Ah ! quel dommage ....
J'aimerais bien t'aider mais je suis novice dans la matière.

Merci pour ton retour.
Anthony
